### DISCUSSION TITLE
[macOS] Bluetooth headset/headphone audio input is activated when running a virtual device

### DISCUSSION THREAD

**Discussion by j7e** [Genymobile/feedback#170] [Genymotion Desktop] [answered]
Two related issues:

When a Bluetooth headphones/headset device is active as the audio output device in macOS, launching and running a virtual device in Genymotion changes the macOS audio input settings by activating the Bluetooth headphones' microphone as the macOS audio input.  This has the effect of lowering the fidelity of the audio output through the headphones by feeding back the microphone input to the headphones.

The problem is temporarily resolved by opening macOS Sound settings and switching the input from the Bluetooth device back to Macbook Pro Microphone each time a virtual device is launched and run or following the virtual device playing audio.

A related issue is that when the virtual device subsequently attempts to play audio (e.g. from a system notification or alert), the headphones' microphone input is re-activated and feeds back through the headphones; however, the macOS audio input setting isn't switched back from Macbook Pro Microphone to the Bluetooth device's microphone.

This issue is temporarily resolved by turning off the Bluetooth headphones and turning them on again. Doing this then switches the macOS audio input to the Bluetooth device, which must then be switched back to the Macbook Pro Microphone. 

Genymotion Desktop v3.7.1 on a 2023 Macbook Pro 16" running macOS 14.5. Bose QC35-II Bluetooth headphones added to macOS as a Bluetooth audio I/O device.

**Comment by jmaneyrol69** [comment, marked as answer]
Thank you for notifying this bug to us @j7e 

We have reproduced on our side too: the main problem is that, when the player launches, it turns on the headset's microphone, which causes Bluetooth headsets to switch the duplex mode. This is the same issue as with Google Emulator (see https://developer.android.com/studio/run/emulator-troubleshooting#emulator-degrades).

The only workaround we have found is opening macOS Sound settings and switching the input from the Bluetooth device back to Macbook Pro Microphone, like you suggested.

A bug fix has been filed to our dev team, and we hope to be able to release a fix in future version of Genymotion Desktop.

If you have any other issues, please contact our support team via our helpcenter at https://support.genymotion.com/hc/en-us/ instead - this board is dedicated to feature requests rather than bug reports.

Thank you for your understanding.
Best regards
